src/runtime/pay-flow-test.js

```diff
@@ -332,6 +338,7 @@ describes.realWin('PayCompleteFlow', {}, env => {
           _client: 'SwG $internalRuntimeVersion$',
           publicationId: 'pub1',
           loginHint: 'test@example.org',
+          productType: undefined,
```

Comment:
is it supposed to be null or undefined or doesn't matter?

Reply:
Here `productType: undefined` is correctly matched.